includes/widgets/video.php

```diff
@@ -243,9 +245,7 @@ protected function _register_controls() {
 				'label' => __( 'Start Time', 'elementor' ),
 				'type' => Controls_Manager::NUMBER,
 				'description' => __( 'Specify a start time (in seconds)', 'elementor' ),
-				'condition' => [
```

Comment:
Is the loop option getting along well with the start and end parameters?

Reply:
From my tests yes but we could give QA to test